codechecker_lib/analysis_manager.py

```diff
@@ -212,7 +212,7 @@ def signal_handler(*arg, **kwarg):
         pool.map_async(check,
                        analyzed_actions,
                        1,
-                       callback=worker_result_handler).get(float('inf'))
+                       callback=worker_result_handler).get()
```

Comment:
I was thinking about this [signal handling comment](https://github.com/Ericsson/codechecker/blob/15d6ae051bc1d3770800ac80193fcde996d60415/codechecker_lib/analysis_manager.py#L198-L202) why it should be kept as it was.